src/compile/mark/encode/tooltip.ts

```diff
@@ -63,17 +63,15 @@ export function tooltip(model: UnitModel, opt: {reactiveGeom?: boolean} = {}) {
   }
 }
 
-export function tooltipRefForEncoding(
+export function tooltipData(
```

Comment:
Instead of splitting this method, how about adding `entryMapper` and `delimiter` as optional params?

Reply:
I doubt that it's cleaner. At least now we can test tooltipData independently. Let me know if you feel strongly about this.